frontend/integration-tests/tests/service-catalog/service-binding.scenario.ts

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable no-undef, no-unused-vars */
+
+import {browser, $, $$, ExpectedConditions as until, by} from 'protractor';
+
+import { appHost, checkLogs, checkErrors, testName } from '../../protractor.conf';
+import * as srvCatalogView from '../../views/service-catalog.view';
+import * as sidenavView from '../../views/sidenav.view';
+import * as crudView from '../../views/crud.view';
+import {execSync} from 'child_process';
+
+describe('Test for Cluster Service Binding', () => {
+  beforeAll(async() => {
+    browser.get(`${appHost}/status/ns/${testName}`);
+    await browser.wait(until.presenceOf($('#sidebar')));
+  });
+
+  afterEach(() => {
+    checkLogs();
+    checkErrors();
+  });
+
+  it('creates a new binding for new service instance `mysql-persistent`', async() => {
+    await sidenavView.clickNavLink(['Service Catalog', 'Service Classes']);
+    await crudView.isLoaded();
+
+    // Filter by service class name to make sure it is on the first page of results.
+    // Otherwise the tests fail since we do virtual scrolling and the element isn't found.
+    await crudView.filterForName('MySQL');
+    await srvCatalogView.cscRowsPresent();
+
+    await srvCatalogView.rowForCSC('MySQL').element(by.linkText('MySQL')).click();
```

Comment:
Any reason not to just use this everywhere?

```js
await element(by.linkText('MySQL')).click();
```

It might fix the failures you're seeing.